drivers/gke/gke_driver.go

```diff
@@ -200,6 +206,120 @@ func (d *Driver) GetDriverCreateOptions(ctx context.Context) (*types.DriverFlags
 		Type:  types.StringType,
 		Usage: "When to performance updates on the nodes, in 24-hour time (e.g. \"19:00\")",
 	}
+	driverFlag.Options["resource-labels"] = &types.Flag{
+		Type:  types.StringSliceType,
+		Usage: "The map of Kubernetes labels (key/value pairs) to be applied to each cluster",
+	}
+	driverFlag.Options["enable-nodepool-autoscaling"] = &types.Flag{
+		Type:  types.BoolType,
+		Usage: "Enable nodepool autoscaling",
+	}
+	driverFlag.Options["min-node-count"] = &types.Flag{
+		Type:  types.IntType,
+		Usage: "Minimmum number of nodes in the NodePool. Must be >= 1 and <= maxNodeCount",
+	}
+	driverFlag.Options["max-node-count"] = &types.Flag{
+		Type:  types.IntType,
+		Usage: "Maximum number of nodes in the NodePool. Must be >= minNodeCount. There has to enough quota to scale up the cluster",
+	}
+	driverFlag.Options["enable-auto-upgrade"] = &types.Flag{
+		Type:  types.BoolType,
+		Usage: "Specifies whether node auto-upgrade is enabled for the node pool",
+	}
+	driverFlag.Options["enable-auto-repair"] = &types.Flag{
+		Type:  types.BoolType,
+		Usage: "Specifies whether the node auto-repair is enabled for the node pool",
+	}
+	driverFlag.Options["local-ssd-count"] = &types.Flag{
+		Type:  types.IntType,
+		Usage: "he number of local SSD disks to be attached to the node",
+	}
+	driverFlag.Options["preemptible"] = &types.Flag{
+		Type:  types.BoolType,
+		Usage: "Whether the nodes are created as preemptible VM instances",
+	}
+	driverFlag.Options["disk-type"] = &types.Flag{
+		Type:  types.StringType,
+		Usage: "Type of the disk attached to each node",
+	}
+	driverFlag.Options["taints"] = &types.Flag{
+		Type:  types.StringSliceType,
+		Usage: "List of kubernetes taints to be applied to each node",
+	}
+	driverFlag.Options["service-account"] = &types.Flag{
+		Type:  types.StringType,
+		Usage: "The Google Cloud Platform Service Account to be used by the node VMs",
+	}
+	driverFlag.Options["oauth-scopes"] = &types.Flag{
+		Type:  types.StringSliceType,
+		Usage: "The set of Google API scopes to be made available on all of the node VMs under the default service account",
+	}
+	driverFlag.Options["master-auth-username"] = &types.Flag{
+		Type:     types.StringType,
+		Usage:    "The username to use for HTTP basic authentication to the master endpoint",
+		Password: true,
```

Comment:
Should this be marked as a password field? Is there a security risk in exposing the username?

Reply:
There is no security risk to expose `userName`, as basic authentication information, I marked it as a password field also, actually, this field cannot be configured by users, because it must be `admin` to make the gke cluster managed successfully by rancher server.

Reply:
If this field cannot be altered then I would think we shouldn't expose it to the user as a config.  It should just be a hardcoded value and we document what we set it to.

Reply:
Updated! Please review it again, thanks.